src/AspNet.Security.OAuth.BattleNet/BattleNetAuthenticationDefaults.cs

```diff
@@ -31,6 +31,22 @@ public static class BattleNetAuthenticationDefaults
     /// </summary>
     public static readonly string CallbackPath = "/signin-battlenet";
 
+    /// <summary>
+    /// Default value for <see cref="OAuthOptions.AuthorizationEndpoint"/>.
+    /// </summary>
+    public static readonly string AuthorizationEndpoint = "https://oauth.battle.net/oauth/authorize";
```

Comment:
Instead of adding new "constants" and deprecating the other ones, we should just use the existing regions to use `oauth.battle.net`? (just in case they'd split that again... 😄)

Reply:
I'll just make this internal and have all the others return this then.

To be honest, I'm not a fan of the existing implementation - a set-only property is weird.

Reply:
> To be honest, I'm not a fan of the existing implementation - a set-only property is weird.

Yeah. If we wanted to make it look less weird, we could probably convert it to an automatic getter+setter property and move the assignation to an `IPostConfigureOptions<BattleNetOptions>` implementation.